sbin/md5/md5.c

```diff
@@ -357,6 +361,15 @@ gnu_check(const char *checksumsfile)
 		rec = malloc(sizeof(*rec));
 		if (rec == NULL)
 			errx(1, "malloc failed");
+
+		if (*filename == '*' ||
+		    *filename == ' ' ||
+		    *filename == 'U' ||
+		    *filename == '^') {
+			if (lstat(filename, &st) != 0)
+				filename++;
+		}
+
```

Comment:
Why did this block move? Note that fileargs_lstat() is available.

Reply:
> Why did this block move? Note that fileargs_lstat() is available.

I had to move it to make exactly one call to fileargs_init() where I need to pass argc & argv.  The latter is recreated after the call to gnu_check() which returns filenames with those one char prefixes.

With this move we're making gnu_check() return a list of sane filenames that we can use.  It's also the place where this block of code makes sense.

I didn't need fileargs_lstat() and it works on symlinks as it is.  It passed the tests.  Do you think I have to change the call?